posthog/src/main/java/com/posthog/PostHogStateless.kt

```diff
@@ -668,5 +705,13 @@ public open class PostHogStateless protected constructor(
         override fun debug(enable: Boolean) {
             shared.debug(enable)
         }
+
+        override fun captureExceptionStateless(
+            throwable: Throwable,
+            properties: Map<String, Any>?,
+            distinctId: String?,
+        ) {
+            shared.captureExceptionStateless(throwable, properties)
```

Comment:
shouldn't this include the distinctId? e.g.

```
shared.captureExceptionStateless(throwable, properties, distinctId)
```

Feels like you should have a test for this too, if one doesn't already exist.